app/frontend/components/pages/dashboard/tabs.tsx

```diff
@@ -1,21 +1,32 @@
+import {MainTabs, SubTabs} from '../../../constants'
 import {h} from 'preact'
 
-export const MainTab = ({name, selectedTab, selectTab, displayName = null}) => {
+type MainTabProps = {
+  name: MainTabs
+  selectTab: (name: MainTabs) => void
+  isSelected: boolean
+  displayName: string | null
+}
+
+export const MainTab = ({name, selectTab, isSelected, displayName = null}: MainTabProps) => {
   return (
-    <li className={`main-tab ${name === 'Staking' ? 'primary' : ''}`}>
+    <li className={`main-tab ${name === MainTabs.STAKING ? 'primary' : ''}`}>
       <input type="radio" id={name} name="tabs" onClick={() => selectTab(name)} />
-      <label className={name === selectedTab ? 'selected' : ''} htmlFor={name}>
+      <label className={isSelected ? 'selected' : ''} htmlFor={name}>
         {displayName || name}
       </label>
     </li>
   )
 }
 
-export const SubTab = ({name, selectedTab, selectTab}) => (
-  <li
-    className={`dashboard-tab ${name === selectedTab ? 'selected' : ''}`}
-    onClick={() => selectTab(name)}
-  >
+type SubTabProps = {
+  name: SubTabs
+  selectTab: (name: SubTabs) => void
+  isSelected: boolean
+}
+
+export const SubTab = ({name, isSelected, selectTab}: SubTabProps) => (
```

Comment:
Switching the order of isSelected and selectTab arguments would be more consistent with MainTab function arguments

Reply:
Fair point :+1: